examples/templates/azure-windows/main.tf

```diff
@@ -82,9 +84,9 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 choco install -y git
 $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-${coder_agent.main.init_script}
+${try(coder_agent.main[0].init_script, "")}
```

Comment:
If this script runs every startup, we should not include one-time commands like installing `choco` and `git`.  Those should get moved into `FirstLogonCommands.xml`, and the `custom_data` is *just* the Coder Agent init script.

Reply:
Ah, sure :)

As an aside, I found FirstLogonCommands very difficult to debug... Where do the error logs go, or how do you get good logs?

Reply:
It seems Windows doesn't directly record the output.

I think what I'll do is have a single Powershell script that is triggered by FirstLogonCommands, and then have this script:

1. Install stuff like chocolatey & git
2. Write the the Coder Agent `init_script` to a local file
3. Register a Task to run the `init_script` on startup

When we invoke this higher-level script, we can also redirect its output to a file.